packages/taquito-local-forging/src/decoder.ts

```diff
@@ -63,6 +64,7 @@ export const decoders: { [key: string]: Decoder } = {
   [CODEC.ZARITH]: zarithDecoder,
   [CODEC.PUBLIC_KEY]: publicKeyDecoder,
   [CODEC.PKH]: pkhDecoder,
+  [CODEC.PKH_ARR]: pkhsDecoder,
```

Comment:
would you consider updating this to PKH_ARRAY or PKH_LIST instead for clarity?

Reply:
At the time, trying to be consistent with existing `PROPOSAL_ARR` but I like your suggestions better

Reply:
if the decoder going with pkhArrayDecoder will update this as PKH_ARRAY.